support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -40,24 +40,27 @@ DomainCreator:
   BinaryCompactObject:
     ObjectA:
       InnerRadius: &ExcisionRadiusA {{ ExcisionRadiusA }}
-      OuterRadius: 6.683
+      OuterRadius: 6.0 #Noticed that if this value is greater than 6.0, we see
+      # constraints blow up at the origin
```

Comment:
The constraints were blowing up cubes around each object, not the origin I though. Also, place the comment on its own line above `OuterRadius`. (same for object B too)